source/unified-test-format/unified-test-format.rst

```diff
@@ -1251,6 +1280,67 @@ expectedCmapEvent
   MUST be an empty document as all fields in this event are non-deterministic.
 
 
+expectedSdamEvent
+`````````````````
+
+The structure of this object is as follows:
+
+.. _expectedEvent_serverDescriptionChangedEvent:
+
+- ``serverDescriptionChangedEvent``: Optional object. Assertions for one or more
+  `ServerDescriptionChangedEvent <../server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events>`__ fields.
+
+  The structure of this object is as follows:
+
+  - ``previousDescription``: Optional object. A value corresponding to the server
+    description as it was before the change that triggered this event.  Test
+    runners MUST follow the rules in `Evaluating Matches`_ when processing this
+    assertion.
+
+  - ``newDescription``: Optional object. A value corresponding to the server
+    description as it was after the change that triggered this event. Test
+    runners MUST follow the rules in `Evaluating Matches`_ when processing this
+    assertion.
+
+  The structure of a server description object (which the ``previousDescription``
+  and ``newDescription`` fields contain) is as follows:
+
+  - ``error``: Optional `expectedError`_ object. One or more assertions for an
+    error expected to be present in the server description.
```

Comment:
Error comparisons are distinct from "Evaluating Matches". That would be another reason I'd propose removing the general application of "Evaluating Matches" to the entire ServerDescription object.